components/toast/toast.jsx

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import UnreadBelowIcon from 'components/widgets/icons/unread_below_icon';
+import CloseIcon from 'components/widgets/icons/close_icon';
+
+function empty() {
+    // do nothing
+}
+
+const someBigNum = 1000;
+
+export default class Toast extends React.PureComponent {
+    static propTypes = {
+        jumpTo: PropTypes.func.isRequired, // required?
+        jumpToMessage: PropTypes.string,
+        jumpFadeOutDelay: PropTypes.number,
```

Comment:
Maybe we could make these more general by just calling them `onClick`, `message`, and `fadeOutDelay`? 0/5

Reply:
the fade one is the only one I wouldn't recomend changing the name, as it doesn't affect if you click on dismiss.